### PR TITLE
[LUP] Add local tasks!

### PR DESCRIPTION
**Pull request by Johnetordoff**
## Purpose

add clause that checks for local tasks and creates a local-dist.py with example task

## Side Effects

Developer happiness

### CODE REVIEW

File: tasks/__init__.py [hunk 1]
```diff
@@ -26,6 +26,12 @@
 NO_TESTS_COLLECTED = 5
 ns = Collection()
 
+try:
+    from tasks import local as local_tasks
```
Comment (pattisdr) [inline tasks/__init__.py]: Are you sure this import works? Maybe you can help me with what I'm doing wrong - 

calling something like `docker-compose run --rm web invoke example_task HELLO`... and getting `No idea what 'example_task' is!` If I move `example_task` into this file, it works fine.

Comment (Johnetordoff) [reply tasks/__init__.py]: there's a suffix so for example task it's `local.example_task' just enter `invoke` for a full list of tasks.

Comment (pattisdr) [reply tasks/__init__.py]: John reminded me that I need to copy the local-dist.py to local -

File: tasks/__init__.py [hunk 2]
```diff
@@ -26,6 +26,12 @@
...
+    ns.add_collection(Collection.from_module(local_tasks), name='local')
+except ImportError:
+    pass
```
Comment (pattisdr) [inline tasks/__init__.py]: @Johnetordoff, do you mind adding a blurb about this to the README (it's purpose, and how to use it) so other folks will know it's there and can use it too?
```bash
cp tasks/local-dist.py tasks/local.py
docker-compose run --rm web invoke local.example_task HELLO
```

File: tasks/__init__.py [hunk 3]
```diff
@@ -17,6 +17,13 @@
 from website import settings
 from .utils import pip_install, bin_prefix
 
+
+try:
+    from tasks import local  # noqa
+except ImportError as error:
+    print('No tasks/local.py file found. '
+          'Did you remember to copy local-dist.py to local.py?')
```
Comment (pattisdr) [inline tasks/__init__.py]: ❤️

**Comment by pattisdr** [comment]
low risk - merging straight to develop

**Pull request merged**